apps/web/app/page.tsx

```diff
@@ -1,3 +1,81 @@
+import Image from "next/image"
+import { Inter, Poppins } from "next/font/google"
+import { RiGitRepositoryLine } from "react-icons/ri";
```

Comment:
cant resolve this dependency

Reply:
![image](https://github.com/keyshade-xyz/keyshade/assets/83924254/e9287be2-ee4a-4bb0-9f6e-f3d5612047c8)